gdal/swig/python/osgeo/gdal.py

```diff
@@ -815,6 +815,8 @@ def DEMProcessingOptions(options=None, colorFilename=None, format=None,
             new_options += ['-combined']
         if multiDirectional:
             new_options += ['-multidirectional']
+        if igor:
```

Comment:
those two lines should be in swig/include/python/gdal_python.i as well. In fact swig/python/osgeo/gdal.py should be regenerated by running "make generate" from swig/python with SWIG 3.0.8

Reply:
@rouault Could you please do it for me? I only wanted to add this algorithm and don't plan to do active development on gdal so it is not worth for me to learn all the gdal project details. And maybe there is much more what I forgot but I'd like that some core developer would polish it. Thanks.

Reply:
I've also never coded in python so I am just guessing whare to add what by greppig `multidirectional` option.

Reply:
OK, I've applied this minor fix and squashed together your commits per 4db55e6

Reply:
Thank you!